test/server.spec.ts

```diff
@@ -255,7 +255,7 @@ describe('POST /chunks', () => {
     expect(reference).toEqual(hash)
 
     const downloadedData = await bee.downloadChunk(reference)
-    expect(downloadedData).toEqual(data)
+    expect(downloadedData.toString()).toEqual(data.toString())
```

Comment:
Are you using the `.toString()` because of some `bee-js` change?

Reply:
TBH I am not sure what caused it, but these tests started to fail.
![Screenshot 2022-06-01 at 17 38 05](https://user-images.githubusercontent.com/7974813/171444160-6d8879a1-8bef-4b1e-97cb-0d44ae372ccd.png)

Reply:
Well that is indeed strange 😳